Sources/Public/DotLottie/Utility/DotLottieManifest.swift

```diff
@@ -0,0 +1,54 @@
+//
+// DotLottieManifest.swift
+// LottieFiles
+//
+// Created by Evandro Harrison Hoffmann on 27/06/2020.
+// Copyright © 2020 LottieFiles. All rights reserved.
+//
+
+import Foundation
+
+/// Manifest model for .lottie File
+struct DotLottieManifest: Codable {
```

Comment:
These are a few files here under `/Sources/Public/DotLottie` that don't have any public declarations. Could you move them to something like `Sources/Private/DotLottie`?